Remotion/Web/Development.WebTesting/WebTestSetUpFixtureHelper.cs

```diff
@@ -107,9 +120,61 @@ private void HostWebApplication ()
       _hostingStrategy.DeployAndStartWebApplication();
     }
 
+    private void VerifyWebApplicationStarted (string webApplicationRoot, TimeSpan applicationPingTimeout)
+    {
+      var stopwatch = Stopwatch.StartNew();
+      var webRequest = (HttpWebRequest) HttpWebRequest.Create (webApplicationRoot);
+      webRequest.Method = WebRequestMethods.Http.Head;
+      webRequest.AllowAutoRedirect = true;
+      HttpStatusCode statusCode = default;
+      Assertion.DebugAssert (statusCode != HttpStatusCode.OK);
+
+      while (statusCode != HttpStatusCode.OK)
+      {
+        try
+        {
+          var responseTask = webRequest.GetResponseAsync();
+          if (responseTask.Wait (applicationPingTimeout - stopwatch.Elapsed))

```

Comment:
EXEC Remotion.ObjectBinding.Web.Development.WebTesting.IntegrationTests.SetUpFixture
[05:13:17] [Exec]    SetUp : System.ArgumentOutOfRangeException : Specified argument was out of the range of valid values.
[05:13:17] [Exec] Parameter name: timeout
[05:13:17] [Exec]    at System.Threading.Tasks.Task.Wait(TimeSpan timeout)
[05:13:17] [Exec]    at Remotion.Web.Development.WebTesting.WebTestSetUpFixtureHelper.VerifyWebApplicationStarted(String webApplicationRoot, TimeSpan applicationPingTimeout) in Remotion\Web\Development.WebTesting\WebTestSetUpFixtureHelper.cs:line 137
[05:13:17] [Exec]    at Remotion.Web.Development.WebTesting.WebTestSetUpFixtureHelper.OnSetUp() inRemotion\Web\Development.WebTesting\WebTestSetUpFixtureHelper.cs:line 102
[05:13:17] [Exec]    at Remotion.ObjectBinding.Web.Development.WebTesting.IntegrationTests.SetUpFixture.SetUp() in Remotion\ObjectBinding\Web.Development.WebTesting.IntegrationTests\SetUpFixture.cs:line 37